lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -85,8 +85,23 @@ class CategoryRenderer {
       const tooltip = this.dom.createChildOf(textEl, 'div', 'tooltip lh-debug');
       tooltip.textContent = audit.result.errorMessage || 'Report error: no audit information';
     } else if (audit.result.explanation) {
-      const explanationEl = this.dom.createChildOf(titleEl, 'div', 'lh-debug');
-      explanationEl.textContent = audit.result.explanation;
+      const explEl = this.dom.createChildOf(titleEl, 'div', 'lh-debug lh-debug--explanation');
+      explEl.textContent = audit.result.explanation;
+    }
+    const warnings = audit.result.warnings;
+    if (!warnings || warnings.length === 0) return auditEl;
+
+    // Add list of warnings or singular warning
+    const warningsEl = this.dom.createChildOf(titleEl, 'div', 'lh-debug lh-debug--warnings');
```

Comment:
do we have any cases of warning + explanation? if so, does it still look okish :)